contracts/Vesting.sol

```diff
@@ -18,38 +18,26 @@ contract Vesting is Ownable {
 
     address public token;
     uint256 public vestingStartDate;
+    uint256 public totalAllocation;
+    uint256 public totalClaimed;
     VestingPlan[] public plans;
 
     mapping(address => uint256) public userPlanId;
     mapping(address => uint256) public allocations;
     mapping(address => uint256) public claimed;
-    uint256 public totalAllocation;
-    uint256 public totalClaimed;
 
-    event AddVestingPlan(
-        uint256 planId,
-        uint256 lockPeriod,
-        uint256 vestingPeriod,
-        uint256 initialUnlockPercent
-    );
+    event AddVestingPlan(uint256 indexed planId, uint256 lockPeriod, uint256 vestingPeriod, uint256 initialUnlockPercent);
```

Comment:
```suggestion
    event VestingPlanAdded(uint256 indexed planId, uint256 lockPeriod, uint256 vestingPeriod, uint256 initialUnlockPercent);
```

Reply:
Doesn't make sense to change `planId` to indexed, it's unique.

Reply:
Good point!